### DISCUSSION TITLE
Sites using swup

### DISCUSSION THREAD

**Discussion by gmrchk** [swup/swup#333] [Show and tell]
> [!NOTE]
> **Swup now has a showcase page with screenshots on the official website!** There you can browse swup-powered sites, get inspired, and submit your own projects. This discussion is kept around as an archive of sorts, but jump over to the website to see up-to-date examples. [Open Showcase Page on Website →](https://swup.js.org/getting-started/showcase/)

You can get an idea of what swup does from the sites that are already using swup. 
If you have your own site to post, please go ahead below. 🚀

**Comment by gmrchk** [comment]
[Dělej, co tě baví - Decathlon](http://delejcotebavi.decathlon.cz/)

**Comment by gmrchk** [comment]
[Plan-k - KPMG](https://www.plan-k.cz/)

**Comment by gmrchk** [comment]
[25 let - Raiffeisen stavební spořitelna](https://www.rsts.cz/25let/)

**Comment by gmrchk** [comment]
[Tvoříme lepší svět - Panasonic](https://plzen.cz.panasonic.com/)

**Comment by gmrchk** [comment]
[Kyle Decker - Personal website](https://kyledecker.me/)

**Comment by gmrchk** [comment]
[FELIX GAUDLITZ](https://felixgaudlitz.com)

**Comment by gmrchk** [comment]
[Karachi Website Design & Web Development Company Pakistan - Pexsol](https://www.pexsol.com/)

**Comment by gmrchk** [comment]
[Metropolis Moving: NYC's Friendliest Moving Professionals - Brooklyn, New York](https://metropolismoving.com/)

**Reply by 23d1** [reply]
[23d.one](https://23d.one)

**Comment by drivebass** [comment]
[Klikkenthéke - Web design inspirational showcase catalogue](https://klikkentheke.com/)

**Comment by vkzawa** [comment]
https://www.reed.senate.gov/

We finished this one last summer but forgot to post it here.

**Reply by gmrchk** [reply]
Oh damn, I love how the header/hero stays on-page and adjusts its size to the content. Really nice showcase of using Swups multiple container functionality the right way. 👏 

I see you're using a custom implementation of `data-swup-selected-if` for navigation active classes. I usually solved it by introducing more tiny swup containers for each nav link, so the active classes were taken care of by the backend implementation basically, but I remember there were some glitches with animations because of that. 

Your implementation seems buggy at times, but it sounds like a great idea for a plugin. It's just a thought, in case you would be interested in sharing it with others yourself or under @swup org.

**Comment by ashehzadgec** [comment]
https://www.nisarsonscaterers.com/

**Comment by currenthaus** [comment]
https://currenthaus.com/

We published this about a year or even more ago, but wanted to include it here.

Absolutely love Swup. We use GSAP with the JS plugin (swupjs) for page transition animations and use events like willReplaceContent to collect animations in the garbage before loading new ones with contentReplaced. pageView is handy for triggering lazy loading images. Keeps the site running fast and smooth.

We also use SwupPreloadPlugin, SwupBodyClassPlugin, and SwupA11YPlugin.

**Reply by k16e** [reply]
So buttery smooth and gorgeous. Love it! ❤️

**Comment by daun** [comment]
[www.elao.com](https://www.elao.com/)

**Comment by folghe92** [comment]
https://grafichefutura.it/

**Comment by wernerglinka** [comment]
For devs using the Metalsmith static site generator, here is a blogpost to help: https://www.glinka.co/blog/metalsmith-website-behaves-like-single-page-app/

tltr... It is super easy ;-)

**Reply by hirasso** [reply]
Very nice! Have you thought about adding the a11y plugin for enhanced accessibility?

**Reply by wernerglinka** [reply]
I’d better reed up on all the plugins. Thank you for the tip!

**Comment by wernerglinka** [comment]
https://www.glinka.co

**Comment by alexandrutocar** [comment]
https://dezea.digital

**Reply by michael-v-bloom** [reply]
Thanks for sharing our new website! Michael (co-founder) from Dezea Studio :)

**Reply by ogerly** [reply]
@michael-v-bloom  your mega cool site has moved me to learn gsap

**Comment by mevanloon** [comment]
https://mediaartfriesland.nl is one I made over the past year, and it uses the new Parallel transition for a cool zoom effect.

**Reply by daun** [reply]
Nice, that's the first parallel animation spotted in the wild 🧃

**Comment by andresclua** [comment]
https://feature.undp.org/from-hope-action-global-goals/ - Built with Astro.

**Comment by andresclua** [comment]
https://valorx.com/ -> built with Astro, Sanity & Netlify.

**Comment by mevanloon** [comment]
I also use the new parallels plugin to simulate an iOS like view transition on my website: https://leobite.nl/

**Comment by adamo** [comment]
https://e-brk.pl/pl built a while ago.

**Comment by daun** [comment]
[Fine Arts Museums of San Francisco](https://www.famsf.org)

**Comment by daun** [comment]
[Gleamer AI](https://www.gleamer.ai/)

**Comment by daun** [comment]
[Herzog & de Meuron](https://www.herzogdemeuron.com/)

**Comment by hirasso** [comment]
[Kulturraum Berlin gGmbH](https://kulturraum.berlin/)

**Comment by daun** [comment]
[Design Campus](https://designcampus.org/)

**Comment by daun** [comment]
[Biennale Architettura 2023](https://labiennale2023.at)

**Comment by daun** [comment]
[Transmediale Kunst](https://transmedialekunst.com/)

**Reply by hirasso** [reply]
Love it!

**Comment by daun** [comment]
[driendl*architects](https://driendl.at)

**Comment by daun** [comment]
[Coop Himmelb(l)au](https://coop-himmelblau.at/)

**Comment by daun** [comment]
[Kunstverein Gartenhaus](https://kunstverein-gartenhaus.com/)

**Comment by daun** [comment]
[Gradient Journal](https://gradient-journal.net/)

**Comment by daun** [comment]
[Halle für Kunst Steiermark](https://halle-fuer-kunst.at/)

**Comment by daun** [comment]
[KOERNOE](https://koernoe.at)

**Comment by hirasso** [comment]
[Kat Petroschkat](https://katpetroschkat.net/) – Artist Website

**Comment by robhealy** [comment]
[Yogamaya](https://yogamaya.com/)

**Comment by daun** [comment]
[Paper Digits](https://paperdigits.nl/)

**Comment by daun** [comment]
[Cyd Stumpel — Creative Developer](https://cydstumpel.nl/)

**Comment by daun** [comment]
[Biennale Arte 2024](https://biennalearte.at/en/)

**Comment by daun** [comment]
[Anton Stallbörger](https://antonstallboerger.com/)

**Comment by hirasso** [comment]
[Luis August Krawen](https://l-a-k.com) – makes use of the new [Parallel Plugin](https://swup.js.org/plugins/parallel-plugin/)

**Comment by k16e** [comment]
https://sayings.cc

Published under a month ago (as of writing) and then added Swup as the best option for page transition, being an Astro SSG site. Have enjoyed the experience of deploying Swup so far with such plugins as Progress, Scroll, and A11y. Still feel there's a few kinks here and there to straighten, but have rather exhausted the docs for options, perhaps I need to switch to the JS plugin option and hand-roll a few more enhancements as I tend to admire on some websites? The https://currenthaus.com/ implementation is just too sweet!

**Comment by hirasso** [comment]
[Hidden Fortress](https://hidden-fortress.com/) – Overlays and filtered lists powered by [Fragment Plugin](https://swup.js.org/plugins/fragment-plugin/)

**Comment by daun** [comment]
[Michael Aerni](https://michaelaerni.ch/)

**Comment by daun** [comment]
[Terra](https://terrahq.com/)

**Comment by daun** [comment]
[Kepler Interactive](https://www.kepler-interactive.com/)

**Comment by daun** [comment]
[Daniel Hopp](https://daniel-hopp.com/)

**Comment by daun** [comment]
[Meat Studio](https://meat.studio/)

**Comment by daun** [comment]
[Andermatt](https://andermatt-realestate.ch/)

**Comment by daun** [comment]
[Fellowship](https://fellowship.xyz)

**Comment by mocodev** [comment]
[https://www.virtualcity.uponor.com/](https://www.virtualcity.uponor.com/)

**Comment by cameronapak** [comment]
https://soulrest.melos.church

**Comment by daun** [comment]
[Kunstmuseum Appenzell](https://kunstmuseum-kunsthalle.ch/)

**Comment by daun** [comment]
[Diplomats](https://diplomats.tv/)

**Comment by daun** [comment]
[Vendredi Society](https://vendredi-society.com/)

**Comment by daun** [comment]
[Ride Out Amsterdam](https://rideout.amsterdam)

**Reply by cydstumpel** [reply]
Thanks for show casing :) Paper Digits is also by me and Stu Stu Studio btw ;)

**Comment by daun** [comment]
[Milk Network](https://milknetwork.com/)

**Comment by hirasso** [comment]
[Studio Spading](https://studiospading.de/)

**Comment by daun** [comment]
[kurzform studio](https://www.kurzform.studio/)

**Comment by daun** [comment]
[MOSQUITO](https://msqt.eu)

**Comment by daun** [comment]
[Jin Su Park](https://jinsupark.com/)

**Comment by daun** [comment]
[Richie Talboy](https://richietalboy.com/)

**Comment by daun** [comment]
[International Programmes in Sustainable Developments](https://ipsd.uni-ak.ac.at/)

**Comment by daun** [comment]
[fjk3 — Raum für zeitgenössische Kunst](https://fjk3.com/)

**Comment by daun** [comment]
[PA Consulting](https://www.paconsulting.com/)

**Comment by daun** [comment]
[Obys Design Education Series](https://des.obys.agency)

**Comment by andresclua** [comment]
[Support Jimmy Lai](https://supportjimmylai.com/) -> built with Wordpress, custom Theme & hosted at Wpengine.

**Comment by hirasso** [comment]
[Templo](https://templo.co.uk/)

**Comment by hirasso** [comment]
[Filter Munich](https://filter-munich.com/)

**Comment by daun** [comment]
[AREA 17](https://area17.com/)

**Comment by k16e** [comment]
[https://farmtech.ng](Farmtech)

**Comment by sanjaysubba** [comment]
https://nokidsinprison.org/

**Reply by hirasso** [reply]
Great work!!

**Comment by hirasso** [comment]
https://www.eduardbodak.com

**Comment by sanjaysubba** [comment]
[Outside](https://outside.studio/)

**Comment by daun** [comment]
[Atelier Andrea Gassner](https://www.atelierandreagassner.at)

**Comment by daun** [comment]
[Breach Tools](https://breachtools.co/)

**Comment by daun** [comment]
[PILOTE PARIS](https://piloteparis.com/)

**Comment by daun** [comment]
[Emma Philip](https://emmaphilip.com)

**Comment by daun** [comment]
[TEATA](https://teata.at/)

**Comment by daun** [comment]
[TQW](https://www.tqw.at)

**Comment by daun** [comment]
[Kunsthalle Wien](https://kunsthallewien.at/)

**Comment by SneakyMouse1** [comment]
[Bosque Nórdico](https://bosquenordico.com/)
